### DISCUSSION TITLE
Hidden Layers

### DISCUSSION THREAD

**Discussion by Jebs88** [keiwando/toonsquid#674] [Q&A] [answered]
I can still select hidden layers. Is this a bug or is there a way to disable this?

**Comment by Cat-Butler** [comment]
Could you please post more information, and maybe a screen recording. Not reproducing it here.

On the canvas? Using which tool?
Hidden using the eye icon or opacity?
Pixel or vector layers?

**Reply by Jebs88** [reply]
![Uploading IMG_2593.png…]()
It occurs when folders are hidden (eye icon : disabled), but the layers within the folders are still visible (eye icon : enabled)

**Reply by keiwando** [reply, marked as answer]
That is a bug. It will be fixed with the 2.0 update. 
Sorry about that!

**Reply by Jebs88** [reply]
> That is a bug. It will be fixed with the 2.0 update. Sorry about that!

I see. No problem. Thank you! 😊
